frontend/src/config/mongolia/layers.json

```diff
@@ -123,13 +125,14 @@
   "hay_harvest": {
     "title": "Hay harvest (soum)",
     "type": "admin_level_data",
-    "path": "https://map.nema.gov.mn/nsodata/DT_NSO_1001_038V3",
-    "requestBody": "title: 'tumee test new server'",    
+    "path": "https://map.nema.gov.mn//api/item",
+    "period":"2021",
     "data_field": "DTVAL_CO",
     "admin_level": 2,
     "admin_code": "CODE",
     "opacity": 0.7,
-    "tblid":"DT_NSO_1001_038V3",
+    "table_id":"DT_NSO_1001_038V3",
+    "requestBody":"tbl_id:DT_NSO_1001_038V3,Period:2021",
```

Comment:
can we not construct this from the `period and table_id`?

Reply:
got it